app/controllers/admin/users_controller.rb

```diff
@@ -78,6 +79,30 @@ def update
       redirect_to admin_user_path(params[:id])
     end
 
+    def reputation_modifier
+      @user = User.find(params[:id])
+      reputation_modifier_value = user_params[:reputation_modifier]
+      note_content = if user_params[:new_note].present?
+                       "Changed user's reputation modifier to #{reputation_modifier_value}. " \
```

Comment:
I suppose these could be in the locales as well?
Also, a bit shorter version (smth like):
```ruby
note_content = "Changed user's reputation modifier to #{reputation_modifier_value}."
note_content += " Reason: #{user_params[:new_note]}" if user_params[:new_note].present?
```

Reply:
I thought about locales, but as this is admin I'm unsure of what language would need to be used? It would be odd for different notes to be added in different languages.